autoload/airline/extensions/tabline/formatters/unique_tail.vim

```diff
@@ -3,6 +3,8 @@
 
 scriptencoding utf-8
 
+let s:fnamecollapse = get(g:, 'airline#extensions#tabline#fnamecollapse', 1)
```

Comment:
please move this down to where it is actually used. Don't need a `s:variable` then.